contribute-docs/content-types/index.md

```diff
@@ -0,0 +1,52 @@
+---
+navigation_title: "Content types"
+description: "Overview of guidelines for choosing the appropriate content types in the Elastic documentation."
+---
+
+# Elastic Docs content types
+
+This page provides an overview of the different content types used in the Elastic documentation and guidelines for choosing the appropriate type when creating new docs pages.
+
+Content types ensure the Elastic Docs are well-structured, consistent and complete. When we use the right content type, we make it easier for users to find the information they need and efficiently complete tasks. 
+
+To help you quickly get started, each content type includes its own guidelines, best practices, and templates.
+
+Before you start drafting a new docs page, identify the appropriate content type for your page.
+Use these guidelines as a framework, not a rulebook. You should adapt structure and syntax where necessary to best serve users, but you should rarely need to deviate from the best practices.
+
+:::{tip}
+Need help choosing a content type or structuring a new page? Reach out to the docs team using the `@elastic/docs` handle in GitHub or post in the [#docs](https://elastic.slack.com/archives/C0JF80CJZ) Slack channel.
+:::
+
+## Mixing different content types
+
+Some documentation pages combine multiple content types.
+
+Mixing different types is fine as long as each section is clearly delineated and serves a distinct purpose. For example, a page about configuring authentication might include:
+
+1. A brief overview of authentication concepts (explanation)
+2. Step-by-step instructions to set up authentication (how-to)
+3. A reference table of authentication settings (reference)
+
+This works because each section is clearly separated and serves a distinct purpose. You shouldn't embed the settings table in the middle of the instructions, or interrupt the steps with conceptual explanations. This would break the flow and make it hard to scan the page for specific information.
+
+When mixing content types, ensure that the overall structure and flow remain clear and logical for users. Use headings and sections to delineate different content types as needed.
+
+:::{note}
```

Comment:
Two ideas:

1. Mark "composable" content types as such in a note in each (for example, how-to)
2. List composable and complete content types in a table here.

Reply:
Can we save this for a follow-up, once we've defined more types? Can comment out this entire section for the moment if necessary.

Reply:
Yes!